cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/StateVariablesAdder.java

```diff
@@ -294,7 +325,8 @@ private void addStatusToCgmes() {
         return nodesFromLines;
     }
 
-    private static Complex complexVoltage(double r, double x, double g, double b,
+    // FIXME(Luma) Consider having this method in LinkData or SV
+    private static Complex calcVoltage(double r, double x, double g, double b,
         double v, double angle, double p, double q) {
         BranchAdmittanceMatrix adm = LinkData.calculateBranchAdmittance(r, x, 1.0, 0.0, 1.0, 0.0,
```

Comment:
@zamarrenolm that is for this kinf of lines of code that I am a bit worried with adding a ````rho```` to dangling lines !